.env

```diff
@@ -0,0 +1,3 @@
+DJANGO_DEBUG=True
+DATABASE_URL=sqlite:///db.sqlite3
+TELEGRAM_TOKEN=1848154316:AAFV7NS30AgBF2jEnwWWDwX9YgMk-obN3mY
```

Comment:
dude wtf

Reply:
@ohld sorry bro, wanted to update my fork, didn't notice that I did the other way around :)